scripts/install/IframePolicyConfigurator.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA
+ */
+
+declare(strict_types=1);
+
+namespace oat\remoteProctoring\scripts\install;
+
+use common_report_Report;
+use oat\oatbox\extension\InstallAction;
+use oat\tao\model\security\Business\Contract\SecuritySettingsRepositoryInterface;
+
+
+class IframePolicyConfigurator extends InstallAction
+{
+    public function __invoke($params)
+    {
+        /** @var SecuritySettingsRepositoryInterface $settingsRepository */
+        $settingsRepository = $this->getServiceManager()->get(SecuritySettingsRepositoryInterface::SERVICE_ID);
+        $securitySettings = $settingsRepository->findAll();
+        $values = ['https://getproctorio.com', ROOT_URL];
+        $securitySettings->findContentSecurityPolicy()->setValue('list');
+        $securitySettings->findContentSecurityPolicyWhitelist()->setValue(implode(PHP_EOL, $values));
+        $settingsRepository->persist($securitySettings);
```

Comment:
```suggestion
        /** @var SecuritySettingsRepositoryInterface $settingsRepository */
        $settingsRepository = $this->getServiceManager()->get(SecuritySettingsRepositoryInterface::SERVICE_ID);
        
        $securitySettings = $settingsRepository->findAll();
        $securitySettings->findContentSecurityPolicy()->setValue('list');
        $securitySettings->findContentSecurityPolicyWhitelist()->setValue(implode(PHP_EOL, ['https://getproctorio.com', ROOT_URL]));
        
        $settingsRepository->persist($securitySettings);
```
I suggest to separate code with new blank lines to separate contexts and increase readability. It takes more time to read when we have all the content wrapped. It is like read a text without paragraphs.